main.cpp

```diff
@@ -261,201 +298,206 @@ struct StereoWidenerAudioPlugin
         int ticksOnSlider = 50;
         std::string sliderColor = "Black";
 
-        void getStateInformation (int sizeInBytes, float inputLevel);
-        void prepareToPlay (double sampleRate, int samplesPerBlock);
-        bool isBypassed (bool customBypassButton, bool nativeBypassButton);
-
+        void getStateInformation( int sizeInBytes, float inputLevel);
+        void prepareToPlay( double sampleRate, int samplesPerBlock);
+        bool isBypassed( bool customBypassButton, bool nativeBypassButton);
     };
 
-    void increaseWetness (MixKnob increase);
-    void decreaseWetness (MixKnob decrease);
-    void smartMute (int audioInput = 1, float audioOutput = 1.1f);
+    void increaseWetness( MixKnob increase);
+    void decreaseWetness( MixKnob decrease);
+    void smartMute( int audioInput = 1, float audioOutput = 1.1f);
 
-    //3 things it can do:
-    // capture audio signal
     void captureAudio();
+    char textInfo();
+    float widenSignal();
+};
 
-    // display text descriptions
-    char textInfo(); //return is displayed for user
+void StereoWidenerAudioPlugin::MixKnob::getStateInformation( int sizeInBytes, float inputLevel)
+{
+    if( sizeInBytes == 256 )
+    {
+        inputLevel = 2.0f;
+    }
+    else
+    {
+        inputLevel = 0.0f;
+    }
+}
+void StereoWidenerAudioPlugin::MixKnob::prepareToPlay( double sampleRate, int samplesPerBlock)
+{
+    if( samplesPerBlock == 44100 )
+    {
+        sampleRate = 2;
+    }
+    else
+    {
+        sampleRate = 0;
+    }
+}
+bool StereoWidenerAudioPlugin::MixKnob::isBypassed( bool customBypassButton, bool nativeBypassButton)
+{
+    if( customBypassButton == true )
+    {
+        nativeBypassButton = false;
+    }
+    else
+    {
+        nativeBypassButton = true;
+    }
+        {
+        return true;
+        }
+}
```

Comment:
Fix this formatting 
why is `return true` in curly braces and indented?
- [ ] revise

Reply:
Because I see/saw the `return` statement as "owned" by `else` rather than `if` or 'if/else'.